opentok/opentok.py

```diff
@@ -475,6 +486,34 @@ def get_archives(self, offset=None, count=None):
         else:
             raise RequestError("An unexpected error occurred", response.status_code)
 
+    def signal(self, session_id, data, connection_id=None):
+        """
+        Send signals to all participants in an active OpenTok session or to a specific client
+        connected to that session.
+        
+        :param String session_id: The session ID of the OpenTok session that receives the signal
+
+        :param Dictionary: Structure that contains both the type and data fields. These correspond
+        to the type and data parameters passed in the client signal received handlers
+
+        :param connection_id String Optional: If it's present the signal is just send to that
```

Comment:
We should say something like this:

The `connection_id` parameter is an optional string used to specify the connection ID of a client connected to the session. If you specify this value, the signal is sent to the specified client. Otherwise, the signal is sent to all clients connected to the session.